cli/tests/pcluster/validators/test_ec2_validators.py

```diff
@@ -708,6 +713,29 @@ def test_capacity_reservation_validator(
     [
         (CapacityReservationInfo({"TotalInstanceCount": 1}), 1, []),
         (CapacityReservationInfo({"TotalInstanceCount": 2}), 1, []),
+        (
+            CapacityReservationInfo(
+                {
+                    "TotalInstanceCount": 0,
+                    "Tags": [
```

Comment:
please add a test with "Tags" but with no CAPACITY_BLOCK_REQUESTED_QUANTITY_TAG_KEY "Key"

Reply:
added and added more unit tests for CapacityReservationInfo methods.